admin/tabs/general/roles/editor/form/RoleFormModel.ts

```diff
@@ -226,4 +263,22 @@ export class RoleFormModel extends HoistModel {
             debounce: 250
         };
     }
+
+    private async lookupDirectoryGroupAsync(directoryGroup: string, recordId: string) {
+        try {
+            const {data} = await XH.fetchJson({
+                autoAbortKey: recordId,
```

Comment:
nice to see this used -- was surprised looking at the FetchService that this abortKey is not actually specific to the the url!  Wondering if the best practice here is to include some more context in this key